ontology/occo.md

```diff
@@ -11,7 +11,7 @@ contact:
   github: zhengj2007
   label: Jie Zheng
   orcid: 0000-0002-2999-0103
-description: The Occupation Ontology (OccO) is an ontology in the domain of human occupations. OccO is an initial ontological presentation of information taken from a standardized occupation taxonomy, the US Standard Occupational Classification (SOC) as enhanced by the O*Net system.
+description: The Occupation Ontology (OccO) is a community-based ontology for occupations. It is designed to facilitate ontological representations of existing occupation standards, such as the US Bureau of Labor Statistics Standard Occupational Classification (US SOC), the International Standard Classification of Occupations (ISCO), the UK National Statistics Standard Occupational Classification (UK SOC), and the European Skills, Competences, Qualifications and Occupations (ESCO) of the European Union 2010. 
```

Comment:
calling an ontology community-based is an anti-pattern. some other thoughts on writing a good description in https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1968

Reply:
Thank you @cthoyt for pointing out the ticket about writing good descriptions